.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,9 +1,8 @@
+
+
 TO CONTRIBUTORS:
 
-Please include a summary of the changes made in this PR above "TO CONTRIBUTORS:", as
-that text will become the commit message. You are also encouraged to append the following
-[co-authorship template](https://help.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors)
-if you'd like to acknowledge suggestions / commits made by other users:
+Please include a summary of the changes made in this PR above "TO CONTRIBUTORS:", as that text will become the commit message. You are also encouraged to append the following [co-authorship template](https://help.github.com/en/github/committing-changes-to-your-project/creating-a-commit-with-multiple-authors) if you'd like to acknowledge suggestions / commits made by other users:
 
 Co-authored-by: name <name@example.com>
```

Comment:
```suggestion
`Co-authored-by: name <name@example.com>`
```
? When the message is formatted in markdown, at least, the `<>` don't appear.

Reply:
This line is intended to by copy-pasted in the PR body.  It won't work (as a `co-authored-by` trailer) if there are backticks around it.